config.lua.dist

```diff
@@ -1,132 +1,235 @@
+-- Combat settings section
+-- In this section, the following settings are defined for the server's combat settings:
+
+-- expFromPlayersLevelRange: the level range for players to gain experience from killing other players.
+-- experienceByKillingPlayers: determines if players can gain experience by killing other players.
+-- hotkeyAimbotEnabled: determines if hotkey aimbot is enabled or not.
+-- killsToBlackSkull: the number of player kills needed before a player's skull changes color to black.
+-- killsToRedSkull: the number of player kills needed before a player's skull changes color to red.
+-- pzLocked: the time in milliseconds that a player must stay in a protection zone.
+-- protectionLevel: the level of protection for players.
+-- removeChargesFromPotions: determines if charges will be removed from potions.
+-- removeChargesFromRunes: determines if charges will be removed from runes.
+-- removeWeaponAmmunition: determines if charges will be removed from ammunition.
+-- removeWeaponCharges: determines if charges will be removed from weapons.
+-- stairJumpExhaustion: the time in milliseconds for a player to become exhausted after jumping up stairs.
+-- timeToDecreaseFrags: the time in seconds for a player's frags to decrease.
+-- whiteSkullTime: the time in seconds for a player's skull to turn white.
+-- worldType: the type of PvP allowed on the server (valid values for worldType are: "pvp", "no-pvp" and "pvp-enforced").
+
+experienceByKillingPlayers = false
+expFromPlayersLevelRange = 75
 hotkeyAimbotEnabled = true
-protectionLevel = 1
-killsToRedSkull = 3
 killsToBlackSkull = 6
+killsToRedSkull = 3
 pzLocked = 60000
-removeChargesFromRunes = true
+protectionLevel = 1
 removeChargesFromPotions = true
+removeChargesFromRunes = true
 removeWeaponAmmunition = true
 removeWeaponCharges = true
+stairJumpExhaustion = 2000
 timeToDecreaseFrags = 24 * 60 * 60
 whiteSkullTime = 15 * 60
-stairJumpExhaustion = 2000
-experienceByKillingPlayers = false
-expFromPlayersLevelRange = 75
+worldType = "pvp"
 
--- Connection Config
--- NOTE: maxPlayers set to 0 means no limit
--- NOTE: allowWalkthrough is only applicable to players
--- NOTE: two-factor auth requires token and timestamp in session key
-ip = "127.0.0.1"
+-- Connection Config section
+-- In this section, the following settings are defined for the server's connection configuration:
+
+-- allowClones: determines if multiple players from the same IP address are allowed to connect.
+-- allowWalkthrough: determines if players can walk through each other.
+-- bindOnlyGlobalAddress: determines if the server should bind only to its global IP address.
+-- enableTwoFactorAuth: determines if two-factor authentication is enabled.
+-- gameProtocolPort: the port number used for the game protocol.
+-- ip: the IP address the server will bind to.
+-- loginProtocolPort: the port number used for the login protocol.
+-- maxPacketsPerSecond: the maximum number of packets that can be sent per second.
```

Comment:
Inbound? Outbound? A lot of this descriptions does not help with anything. Some of them are even wrong ☹️

Reply:
Could you suggest fixes instead of just saying there's something wrong so I can fix it.